test/test_printing.py

```diff
@@ -51,8 +51,8 @@ def test_printing_snapshot(default_asset_id, printing_mocks):
 
 
 export_image_task_mock_data = [
-    ("50%", "active", "creating the image", get_export_image_task_mock_data(False)),
-    ("n/a", "completed", "n/a", get_export_image_task_mock_data(True))
+    ("50%", "active", "creating the image", get_export_image_task_mock_data(True)),
+    ("n/a", "completed", "n/a", get_export_image_task_mock_data(False))
```

Comment:
I hope this correct, because true and false changed

Reply:
with true and false,I suggest usually enums or keyword parameter

Reply:
keyword parameters are a good idea! fixed